enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -16,10 +19,18 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${aceEditorStyles}
   ${saveDomImageStyles}
   ${({ theme }) => getRootStyle(theme)}
+  ${({ theme }) => getWrapperStyle(theme)}
 
   ${({ baseUrl }) => defaultFontFiles({ baseUrl })}
 
   svg {
     display: inline;
   }
 `;
+
+const getFontFamily = (theme: EmbeddingTheme) =>
+  theme.fontFamily ?? DEFAULT_FONT;
+
+const getWrapperStyle = (theme: EmbeddingTheme) => css`
+  font-size: ${theme.other.fontSize ?? "0.875em"};
+`;
```

Comment:
The default `theme` is from our global override. We might not want to do a global re-declaration becausethe Embedding SDK theme options in `theme.other` would leak out to the main app. Defining the `theme` type explicit does the trick.